src/mxcp/server/core/config/models.py

```diff
@@ -392,11 +392,12 @@ def _apply_defaults(self) -> UserAuthConfigModel:
 class UserModelConfigModel(BaseModel):
     model_config = ConfigDict(extra="forbid", frozen=True)
 
-    type: Literal["claude", "openai"]
+    type: Literal["anthropic", "openai"]
     api_key: str | None = None
     base_url: str | None = None
     timeout: int | None = None
     max_retries: int | None = None
+    options: dict[str, Any] = Field(default_factory=dict)
```

Comment:
This means we have no control or way to document or validate our own config... We must decide which fields we support and document them, and to follow the existing patterns, it should be a different set of configs per model.